src/configurer/routingutils/RIPConfigXMLGenerator.py

```diff
@@ -0,0 +1,96 @@
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from model.routing.RIPConfig import RIPConfig
+    from model.devices.Router import Router
+
+class RIPConfigXMLGenerator:
```

Comment:
imo this class is not needed if every method is static, it only complicates things and makes code longer, usage of

```python
from configurer.routingutils.RIPConfigXMLGenerator import RIPConfigXMLGenerator

RIPConfigXMLGenerator.generate(...)
```

isn't imo any better than

```python
import configurer.routingutils.rip as riputils

riputils.generateXML(...)
```